package.json

```diff
@@ -35,6 +35,8 @@
     "multer": "^1.4.2",
     "opentracing": "^0.14.4",
     "pino": "^6.0.0",
+    "platform": "^1.3.6",
+    "project-lebanon": "file:web-app",
```

Comment:
@nicholasmorgan :smile: 

Please dont use npm i web-app because it adds it as dependency to the node package.json

cd web-app && npm i is better

should probably add npm script to make the install just one command

Reply:
Oops! Should have thought about that rather than just copy paste instructions